### PR TITLE
Avatar's move_state informs animation and velocity

### PR DESCRIPTION
**Pull request by jrslagle**

### CODE REVIEW

**Comment by Master-Foo** [comment]
I'll take a look at it now.

File: source/entities/avatar/view.py [hunk 1]
```diff
@@ -1,12 +1,5 @@
-import pygame
-
-from source.action import action
 from source.abstract.entities.human import view
 
-from animation_config import run
-from animation_config import stand
-from animation_config import walk
-
 class View(view.View):
```
Comment (jrslagle) [inline source/entities/avatar/view.py]: These were just dead code I found.

**Comment by Master-Foo** [comment]
Approved!

**Comment by jrslagle** [comment]
Woo! I did one!

**Comment by Master-Foo** [comment]
See how easy it is to get approved when you commit clean code?

**Comment by jrslagle** [comment]
Yea, it's easy when you do it right. Very round about when you're figuring it all out.

**Comment by Master-Foo** [comment]
The general work flow is, when you work on an issue, make a branch for that issue. Only work on that issue within that branch. If you want to work on a different issue, make a new branch. Only work on that issue in that branch. When an issue is ready to be committed, you only commit that branch and not the other branch. That way, you aren't submitting FrankenCode.

**Comment by Master-Foo** [comment]
Now, go try making whatever vegetable I assigned to you. I gave that to you specifically so you could have something to learn the ropes better.

Reference My Corn and @fishscene Tomato for examples.

**Comment by Master-Foo** [comment]
Also, take a look at my electrical grid system. It's primitive, but cool. We'll be using it a lot.

**Pull request merged**